lib/siftsciex/verification/response.ex

```diff
@@ -24,7 +24,7 @@ defmodule Siftsciex.Verification.Response do
         status: 50
       }
   """
-  @spec process(String.t, String.t) :: __MODULE__.t
+  @spec process(String.t, String.t) :: CheckResponse.t() | SendResponse.t()
```

Comment:
:+1:  Thanks

Reply:
Funny fact: this will not compile on newer versions of Elixir!